code/ui/generalhud/voicechat/VoiceChatEntry.cs

```diff
@@ -68,26 +80,6 @@ public override void Tick()
 
             _voiceLevel = _voiceLevel.LerpTo(_targetVoiceLevel, Time.Delta * 40.0f);
 
-            Style.Left = 30f + _voiceLevel * -30.0f * timeoutInv;
-            Style.BackgroundColor = null;
-
-            if (Client != null && Client.IsValid() && Client.Pawn is TTTPlayer player)
-            {
-                if (player.IsSpeaking)
-                {
-                    Style.BackgroundColor = Color.Black.WithAlpha(0.9f);
-                }
-
-                if (player.LifeState == LifeState.Dead)
-                {
-                    Style.BackgroundColor = _deadColor.WithAlpha(0.9f);
-                }
-                else if (player.IsTeamVoiceChatEnabled && player.Role is not NoneRole)
-                {
-                    Style.BackgroundColor = player.Role.Color.WithAlpha(0.9f);
-                }
-            }
-
```

Comment:
Did you removed the player state indicators?

Reply:
I had reworked this part when fixing a bug and forgot to add the spectator colors.